packages/opentelemetry-metrics/src/export/Controller.ts

```diff
@@ -37,7 +37,7 @@ export class PushController extends Controller {
   ) {
     super();
     this._timer = setInterval(() => {
-      this._collect();
+      this._collect().catch(_err => {});
```

Comment:
Is it intended to swallow all errors?

Reply:
Good question, it seemed to have been swallowing them before if I'm not mistaken, if so this retains that behaviour, I was not sure if we wanted to throw and I don't think there is an available logger at this point, unless I can use console

Reply:
`somePromise.catch((err) => {throw err})` is the same as `somePromise`. throwing immediately does nothing but fix the lint rule. The idea behind no hanging promises rule is that you prevent having a promise which rejects but is not handled. If you rethrow, it just becomes an unhandled promise rejection again.

Reply:
right I guess throw would retain the same behaviour, which would only fix the lint rule. though I do not have enough context on the various unhandled promises so determine how to handle them. 
I think for this, if a collection fails, its worth and error log, but I don't think a logger is available here. Can we use `console.error` ?

Reply:
We can't use console.error because that would remove the ability of the user to control/redirect/suppress logging. In my opinion if we are only suppressing the lint rule, it would be better to do so with an eslint comment which makes it clear that we are suppressing the lint rule intentionally rather than some odd-looking piece of code which effectively is no-op.

Reply:
yes, good idea!